src/license/src/feature.rs

```diff
@@ -46,7 +46,6 @@ macro_rules! for_all_features {
             { TestPaid,             Paid,       "A dummy feature that's only available on paid tier for testing purposes." },
             { TimeTravel,           Paid,       "Query historical data within the retention period."},
             { GlueSchemaRegistry,   Paid,       "Use Schema Registry from AWS Glue rather than Confluent." },
-            { SnowflakeSink,        Paid,       "Delivering data to SnowFlake." },
```

Comment:
Do we still need to set sw sink as a paid feature

Reply:
Think twice, we may keep it paid, because user does not aware of this change. cc @hzxa21  for decision.